src/sparseml/modifiers/distillation/output/pytorch.py

```diff
@@ -37,6 +38,8 @@ def on_initialize(self, state: State, **kwargs) -> bool:
             return False
 
         self.wrappers_ = {}
+        if kwargs.get("fsdp_active"):
+            self.fsdp_active_ = True
```

Comment:
what do we want to convey with the trailing `_` in the variable names? if private our repos generally standardize to putting the `_` first.

Reply:
Its meant to convey a private variable. Since the `Module` class inherits from pydantics `BaseModel`, any class variables led by an `_` are readonly. Not sure if there is a better way to get around this? I'm using the trailing _ in a lot of the modifiers to signify "private" variables